lib/spells/effects/RemoveObstacle.cpp

```diff
@@ -32,15 +32,12 @@ namespace effects
 VCMI_REGISTER_SPELL_EFFECT(RemoveObstacle, EFFECT_NAME);
 
 RemoveObstacle::RemoveObstacle()
-	: LocationEffect(),
-	removeAbsolute(false),
-	removeUsual(false),
-	removeAllSpells(false)
+	: removeAbsolute(false),
```

Comment:
Weird formatting. Might need change in clang-format but please use either
```
RemoveObstacle::RemoveObstacle()
	: removeAbsolute(false)
	, removeUsual(false)
```
or
```
RemoveObstacle::RemoveObstacle():
	removeAbsolute(false),
	removeUsual(false)
```

Reply:
Okay. I prefer the latter actually, but this formatting was there before clang-tidy was modernized it.

Reply:
@IvanSavenko did all the latter style